src/components/Comments.jsx

```diff
@@ -50,6 +66,17 @@ const Comments = ({ article_id, comments, setComments }) => {
                   <strong>Created: </strong>
                   {comment.created_at}
                 </h4>
+                <div>
+                  {comment.author === user ? (
+                    <button
+                      onClick={(event) =>
+                        handleDeleteComment(event, comment.comment_id)
+                      }
+                    >
+                      Delete
+                    </button>
+                  ) : null}
+                </div>
```

Comment:
There seem to be several `<div>` elements creeping into your code, can you think of more appropriate semantic html elements you could use instead?